lib/opensusebasetest.pm

```diff
@@ -662,7 +662,19 @@ sub post_fail_hook {
             record_info("no $program", "Could not find '$program' on the system", result => 'fail') && die "$program does not exist on the system";
         }
     }
-
+    
+    if (get_var('FULL_LVM_ENCRYPT')) {
```

Comment:
With such condition it won't be triggered for `lvm_thin_check.pm` scenario, as we have `LVM_THIN_LV` there only.